demisto_sdk/commands/generate_unit_tests/tests/generate_unit_tests_test.py

```diff
@@ -0,0 +1,79 @@
+import ast
+import itertools
+import os
+import pytest
+from pathlib import Path
+from ast import parse
+from demisto_sdk.commands.common.legacy_git_tools import git_path
+from demisto_sdk.commands.generate_unit_tests.generate_unit_tests import run_generate_unit_tests
+
+ARGS = [({'use_demisto': False}, 'malwarebazaar_all.txt'),
+        ({'use_demisto': False, 'commands': 'malwarebazaar-comment-add'}, 'malwarebazaar_specific_command.txt')]
+
+
+def compare_ast(node1, node2):
+    """
+     Recursively comparing ast objects.
+    """
+    if type(node1) is not type(node2):
+        return False
+    if isinstance(node1, ast.AST):
+        for (k, v) in vars(node1).items():
+            if k in ('lineno', 'col_offset', 'ctx'):
+                continue
+            if not compare_ast(v, getattr(node2, k)):
+                return False
+        return True
+    elif isinstance(node1, list):
+        return all(itertools.starmap(compare_ast, zip(node1, node2)))
+    else:
+        return node1 == node2
+
+
+class TestUnitTestsGenerator:
+    test_files_path = Path(git_path(), 'demisto_sdk', 'commands', 'generate_unit_tests', 'tests', 'test_files')
```

Comment:
I think this should do the job
```suggestion
    test_files_path = Path(os.getcwd()) / 'test_files'
```